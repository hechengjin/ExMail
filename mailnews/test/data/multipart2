Content-Type: multipart/mixed; boundary=boundary

This is a text/html message. This part shouldn't appear at all!

--boundary
Content-Type: text/html

<html><body>Multipart HTML message with just a single part!
</body></html>

--boundary--

Actually, this part shouldn't appear either.
