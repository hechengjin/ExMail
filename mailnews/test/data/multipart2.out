<html><body>Multipart HTML message with just a single part!
</body></html>